hubblestack/extmods/fdg/curl.py

```diff
@@ -14,20 +14,17 @@
 endpoints.
 """
 from __future__ import absolute_import
-import json
 import logging
 import requests
 
-from salt.exceptions import CommandExecutionError
 
-log = logging.getLogger(__name__)
+LOG = logging.getLogger(__name__)
 
 
 def request(url,
             function='GET',
             params=None,
             data=None,
-            headers=None,
```

Comment:
Oh, alright, sorry, I thought the functionality changed and since it was not used I thought there's no need for it anymore. I'll rebase with the complete function